app/uk/gov/hmrc/cgtpropertydisposals/connectors/SubscriptionConnector.scala

```diff
@@ -43,20 +44,56 @@ class SubscriptionConnectorImpl @Inject()(http: HttpClient, val config: Services
 
   val baseUrl: String = config.baseUrl("subscription")
 
-  val url: String = s"$baseUrl/subscribe/individual"
+  val url: String = s"$baseUrl/subscribe"
 
   override def subscribe(
     subscriptionDetails: SubscriptionDetails
-  )(implicit hc: HeaderCarrier): EitherT[Future, Error, HttpResponse] =
+  )(implicit hc: HeaderCarrier): EitherT[Future, Error, HttpResponse] = {
+    val subscriptionRequest = SubscriptionRequest(
+      subscriptionDetails.contactName.fold[TypeOfPerson](_ => TypeOfPerson.Trustee, _ => TypeOfPerson.Individual),
+      subscriptionDetails.contactName.fold(_.value, n => s"${n.firstName} ${n.lastName}"),
+      subscriptionDetails.emailAddress,
+      subscriptionDetails.address,
+      subscriptionDetails.sapNumber
+    )
+
     EitherT[Future, Error, HttpResponse](
       http
-        .post(url, Json.toJson(subscriptionDetails), headers)(
+        .post(url, Json.toJson(subscriptionRequest), headers)(
           implicitly[Writes[JsValue]],
           hc.copy(authorization = None),
           ec
         )
         .map(Right(_))
         .recover { case e => Left(Error(e)) }
     )
+  }
+
+}
+
+object SubscriptionConnectorImpl {
+
+  sealed trait TypeOfPerson { val value: Int }
+
+  object TypeOfPerson {
+
```

Comment:
I think a better name for this is AffinityType

Reply:
OK, this probably makes sense to keep it as TypeOfPerson if DES are expecting that field name in the JSON request.